FaceDetection/faceClassifer.py

```diff
@@ -67,7 +72,7 @@ class FaceClassifer():
     """
     def __init__(self, num_classes, label_encoder):
         self.num_classes = num_classes
-        self.device = torch.device("cuda" if torch.cuda.is_available() else "cpu")
+        self.device = torch.device("cuda:1" if torch.cuda.is_available() else "cpu")
```

Comment:
Let's consider using the device specified in the arguments and not hard-code it here.